start-mycroft.sh

```diff
@@ -105,7 +107,7 @@ function launch-all() {
     launch-background skills
     launch-background audio
     launch-background voice
-    launch-background enclosure
+    launch-background gui
```

Comment:
Is this intentionally replacing enclosure with gui rather than adding another line?

Reply:
yes its intentional, the aim is that enclosure will be fully deprecated, the generic enclosure only purpose was to launch the gui protocol, with thi change launching enclosure is doing nothing except take up resources, it just sits there doing nothing...

the only circumstance you want to launch enclosure is in a mk1, and we generally do not encourage using these scripts but rather to create systemd services or equivalent